datahub-web-react/src/app/identity/user/UserListItem.tsx

```diff
@@ -121,16 +121,23 @@ export default function UserListItem({ user, canManageUserCredentials, selectRol
                     trigger={['click']}
                     overlay={
                         <Menu>
-                            <Menu.Item disabled={!shouldShowPasswordReset} onClick={() => setIsViewingResetToken(true)}>
-                                <UnlockOutlined /> &nbsp; Reset user password
+                            <Menu.Item
+                                disabled={!shouldShowPasswordReset}
+                                onClick={() => setIsViewingResetToken(true)}
+                                data-testid="resetButtonClick"
```

Comment:
`rest-button-item` instead please. `Click` is a action. We could do other actions here instead of click too